login.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+     include_once('header.php'); // Include Header once
+     
+if (isset($_POST['login_user'])) {
+    $errors = array();
+    $first_name = '';
+    $email_address = mysqli_real_escape_string($db_connect, $_POST['email_address']);
+    $user_password = mysqli_real_escape_string($db_connect, $_POST['user_password']);
+    
+    if (empty($email_address)) {
+          array_push($errors, 'Email address is required');
+    }
+    if (empty($user_password)) {
+        array_push($errors, 'Password is required');
+    }
+    if (count($errors) == 0) {
+        $user_password = md5($user_password);
```

Comment:
Let's not use `md5()`.
There is a better way to hash and verify the password in PHP:
[password_hash()](https://www.php.net/manual/en/function.password-hash.php)
[password_verify()](https://www.php.net/manual/en/function.password-verify.php)